pkg/subproxy/subproxy.go

```diff
@@ -221,10 +221,10 @@ func (s *subscriberProxy) addSimObjectToSite(site *models.OnfEnterprise_Enterpri
 		updStr = fmt.Sprintf("sim-card[sim-id=%s]/description", *sim.SimId)
 		updates = gnmiclient.AddUpdate(updates, gnmiclient.UpdateString(updStr, s.AetherConfigTarget, sim.Description))
 		updStr = fmt.Sprintf("sim-card[sim-id=%s]/iccid", *sim.SimId)
-		updates = gnmiclient.AddUpdate(updates, gnmiclient.UpdateUInt64(updStr, s.AetherConfigTarget, sim.Iccid))
+		updates = gnmiclient.AddUpdate(updates, gnmiclient.UpdateString(updStr, s.AetherConfigTarget, sim.Iccid))
 	}
 
-	iccid := uint64(1234)                   //TODO see how we can get this value
+	iccid := ""                             //TODO see how we can get this value
```

Comment:
If we don't have the value, can we leave the ICCID unset instead of setting it to the empty string? My concern is that it's set to "unique" in the modeling, and it might consider the empty string to be a value and prevent two sim card from both being set to the empty string (but wouldn't prevent two of them from being unset).

Reply:
Done not setting the iccid